### ISSUE TITLE
Latest wax + IE8: "Error: 'target.className' is null or not an object"

### ISSUE THREAD

**Issue by setvik** [cutting-room-floor/wax#172]
System Info:
Windows XP
IE 8

Clicking a map in the latest wax (5.0.0-alpha2) on IE8 results in the following error in wax.mm.js on line 2770 when using interactions:

"Error: 'target.className' is null or not an object"

Unfortunately the error blocks further js execution, making the latest wax/modestmaps unusable in ie8.

The error is reproducible with the following code:

``` html
<html>
<head>
  <script src='wax/ext/modestmaps.min.js' type='text/javascript'></script>
  <script src='wax/dist/wax.mm.js' type='text/javascript'></script>
  <link href='wax/theme/controls.css' rel='stylesheet' type='text/css' />
</head>
<body>
<div id='intro-map'></div>
<script>

var mm = com.modestmaps;
var url = 'http://api.tiles.mapbox.com/v3/mapbox.geography-class.jsonp';

wax.tilejson(url, function(tilejson) {
  var m = new mm.Map('intro-map',
    new wax.mm.connector(tilejson),
    new mm.Point(700,400));

  m.setCenterZoom(new mm.Location(tilejson.center[1],
    tilejson.center[0]),
    tilejson.center[2] - 3);

  wax.mm.zoomer(m).appendTo(m.parent);
  wax.mm.interaction(m);
});

</script>  
</body>
</html>
```

**Comment by tmcw** [comment]
Fixed in 1144af895719c152f90241e50da9ce2e4a9077c5 - try current master, and you'll need to update to modestmaps 1.0.0-alpha2

**Comment by tmcw** [comment]
Fixed in the current tag.
